src/main/scala/pl/iterators/sealedmonad/syntax/SealedSyntax.scala

```diff
@@ -23,7 +23,43 @@ final class SealedFAOps[F[_], A](private val self: F[A]) extends AnyVal {
 }
 
 final class SealedFOptAOps[F[_], A](private val self: F[Option[A]]) extends AnyVal {
-  def valueOr[ADT](orElse: => ADT): Sealed[F, A, ADT]     = Sealed.valueOr(self, orElse)
+
+  /** Returns a Sealed instance containing value `A` if it's present, else short-circuits with given `ADT` value.
```

Comment:
Not sure if "present" is the best wording? Maybe defined as it's called in Option?